tests/instruments/lecroy/test_lecroyT3DSO1204_with_device.py

```diff
@@ -389,7 +396,7 @@ def test_download_data_sparsing(self, instrument):
         from matplotlib import pyplot as plt
 
         instrument.ch_1.display = True
-        instrument.single()
+        instrument.arm_acquisition()
```

Comment:
I'm seeing this change a lot. This method name was used in another instrument already for the same thing (get a single measurement) -- should we maybe harmonize that and use `single()` also for this instrument? 
(afaict we have not included this instrument in a release yet, so we don't even need to deprecate anything as we don't break backwards compatibility)

Reply:
The reason I have renamed `single` into `arm_acquisition` is because that is how that function is called in the documentation. But I have nothing against renaming back to `single`.

Reply:
That is a good reason. However, we are slowly starting to nudge towards a more homogeneous API across similar instrument classes, and for that shared names for the same things are preferable.
@bmoneke thoughts?

Reply:
I'm for unification of names as well.

Is it similar to a "single" button on oscilloscopes?

Then I'd use "single".

Reply:
I renamed the `arm_acquisition` method into `single` in commit eff7dfb2adde78103cf6b4825d16ea14ee29daf0

Reply:
@bmoneke 

> Is it similar to a "single" button on oscilloscopes?

Yes

Reply:
@bilderbuchi . If this conversation is resolved for you, please merge.